SimPEG/EM/NSEM/Utils/dataUtils.py

```diff
@@ -54,7 +54,114 @@ def rotateData(NSEMdata, rotAngle):
 
     return Data.fromRecArray(outRec)
 
+# Function to get data and data info
+def extract_data_info(NSEMdata):
```

Comment:
if you have an example or test that uses this and the following util, that would be helpful in terms of seeing usage